packages/web/components/Carousel/Track.tsx

```diff
@@ -139,8 +141,8 @@ export const Track: React.FC = ({ children }) => {
         <VStack ref={node} spacing={5} alignItems="stretch">
           <MotionFlex
             dragConstraints={node}
-            onDragStart={handleDragStart}
-            onDragEnd={handleDragEnd}
+            onDragStart={isMobile ? undefined : handleDragStart}
```

Comment:
I it is an improvement but it is still super easy to skip to the next step by accident. The impact of that is less so now after you've fixed the persistence in the inputs but it's still pretty disorientating. Is it possible to make it so the active drag area is the same as the visible portion of the next step?

Also, in my changes I didn't add a check for the prev/next buttons when at the start/end of the track.

Reply:
how is it super easy to skip to the next step by accident? isn't the only way one could do that by pressing the tiny edge of the next quest?

Reply:
>  Is it possible to make it so the active drag area is the same as the visible portion of the next step?

but it is like that atm, isn't it? one can't drag the current quest.

> Also, in my changes I didn't add a check for the prev/next buttons when at the start/end of the track. 

will add this 👌

Reply:
> how is it super easy to skip to the next step by accident? 

Actually, giving this another go today it seems fine.

Reply:
:D pls give it another look, i hid the relevant buttons on first/last step